dispatch/static/manager/src/js/components/ArticleEditor/index.js

```diff
@@ -22,6 +23,8 @@ class ArticleEditorComponent extends React.Component {
     super(props)
 
     this.toggleStyle = this.toggleStyle.bind(this)
+
+    this.state = { isSaved: true }
```

Comment:
@joncppl Do you mind if I move the `isSaved` boolean to the store? I think it can be a default attribute for all of the `single` item stores, and should be pretty easy to update via the reducers.